app/app.go

```diff
@@ -706,6 +689,15 @@ func NewExocoreApp(
 		cast.ToString(appOpts.Get(srvflags.EVMTracer)),
 		app.GetSubspace(evmtypes.ModuleName),
 	)
+	// the AVS manager keeper is the AVS registry. it allows registered operators to add or
+	// remove AVSs.
```

Comment:
please add the comment here to indicate that this keeper is initialized after the EVM keeper because it depends on the EVM keeper.